content/docs/pipelines/pipelines-tutorial.md

```diff
@@ -0,0 +1,616 @@
+
+Run the following Cloud SDK command to install `kubectl`:
+
+```
+gcloud components install kubectl
+```
+
+### Set up some handy environment variables
+
+Set up the following environment variables for use throughout the tutorial:
+
+1. Set your GCP project ID. In the command below, replace `<YOUR-PROJECT-ID>` 
+  with your [project ID][gcp-project-id]:
+
+    ```
+    export PROJECT=<YOUR-PROJECT-ID>
+    gcloud config set project ${PROJECT}
+    ```
+
+1. Set the zone for your GCP configuration. Choose a zone that offers the
+  resources you need. See the guide to [GCP regions and zones][regions-zones].
+    * Ensure you have enough Compute Engine regional capacity.
+      By default, the GKE cluster setup described in this guide
+      requires 16 CPUs.
+    * If you want a GPU, ensure your zone offers GPUs.
+
+    For example, the following commands set the zone to `us-central1-c`:
+
+    ```
+    export ZONE=us-central1-c
+    gcloud config set compute/zone ${ZONE}
+    ```
+
+1. If you want a custom name for your Kubeflow deployment, set the 
+   `DEPLOYMENT_NAME` environment variable. The deployment name must be 
+   **4-20 characters** in length. Note that the name must be the same 
+   as the one you use in later steps of this tutorial when configuring the 
+   **redirect URI** for the OAuth client credentials. If you don't set this 
+   environment variable, your deployment gets the default name of `kubeflow`:
+
+    ```
+    export DEPLOYMENT_NAME=kubeflow
+    ```
+
+### Deploy Kubeflow
+
+Deploy Kubeflow on GCP:
+
+1. Follow the instructions in the 
+  [GKE getting-started guide](/docs/started/getting-started-gke), 
+  taking note of the following:
+
+  * Set up **OAuth client credentials** and **Cloud Identity-Aware Proxy (IAP)**
+    as prompted during the deployment process. So, **do not choose the deployment 
+    option to skip IAP**. IAP ensures you can connect securely to the Kubeflow
+    web applications.
+  * When setting up the **authorized redirect URI** for the **OAuth client 
+    credentials**, use the same value for the `<deployment_name>` as you used
+    when setting up the `DEPLOYMENT_NAME` environment variable earlier in this
+    tutorial.
+  * Use the Kubeflow **deployment UI** as a quick way to set up a Kubeflow 
+    deployment on GCP. The getting-started guide describes how to use the
+    deployment UI. If you want more control over the configuration of your
+    deployment you can use the `kfctl.sh` script instead of the UI. The script
+    is also described in the getting-started guide.
+  * Choose **Kubeflow version v0.4.1** or later.
+
+    The following screenshot shows the Kubeflow deployment UI with hints about
+    the value for each input field:
+
+    <img src="/docs/images/gcp-e2e-deploy-kubeflow.png" 
+        alt="Prediction UI"
+        class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. (Optional) Use `kubectl` to connect to your cluster:
```

Comment:
Maybe describe why they'd want to do this if it's optional (is this just for debugging purposes? watch the cluster spin up?)

Reply:
It's to give them something do while waiting for the UI. I've added this text:
"If you want to examine your cluster while waiting for the UI to
   be available..."